includes/Classifai/Plugin.php

```diff
@@ -42,6 +52,10 @@ public function enable() {
 		add_filter( 'plugin_action_links_' . CLASSIFAI_PLUGIN_BASENAME, [ $this, 'filter_plugin_action_links' ] );
 		add_filter( 'robots_txt', [ $this, 'maybe_block_ai_crawlers' ] );
 		add_action( 'after_classifai_init', [ $this, 'load_action_scheduler' ] );
+
+		// Initialize the MCP adapter so the hooks are loaded.
+		$this->mcp_adapter = McpAdapter::instance();
```

Comment:
In my testing, this was needed to ensure the MCP code was loaded. Not ideal but I'm assuming this will get better as this package matures